espressif/vscode-esp-idf-extension
# update dockerfile devcontainer

## Description

Update Dockerfile and devcontainer.json to latest IDF and python version.

Add Python Lib directory to Path in python env install.

Fixes #1100
Fixes #1101
Fixes #1106

## Type of change

Please delete options that are not relevant.
- Bug fix (non-breaking change which fixes an issue)

## Steps to test this pull request

Provide a list of steps to test changes in this PR and required output

1. Click on "Configure ESP-IDF extension" command
2. Execute setup action
3. Observe results. Python libraries used in setup should come from embed Python.

- Expected behaviour:

- Expected output:

## How has this been tested?

Manual test of setup wizard on Windows.

**Test Configuration**:
* ESP-IDF Version: 5.3
* OS (Windows,Linux and macOS): Windows


## Checklist
